source/css/scss/app/loancards-2.scss

```diff
@@ -117,6 +122,17 @@ $loan-card-border: 1px solid $kiva-stroke-gray;
 		font-weight: 300;
 	}
 
+	.loan-card-2-loan-length {
+		font-size: rem-calc(14);
```

Comment:
We have the defined class of "small-text" that you can set to this HTML element to achieve this css rule.

Reply:
@BoulderBrains Great tip/reminder. Thanks!